samples/sample_encode/include/pipeline_encode.h

```diff
@@ -184,6 +184,8 @@ struct sInputParams
     mfxU16 AdaptiveB;
     mfxU32 nMaxFrameSize;
 
+    mfxU16 BitrateLimit;
```

Comment:
Perhaps it's a time to introduce "Tri-state" as a type on samples level? Not in this commit, of course.
@dvrogozh ?

Reply:
I am not sure I understand what you mean. PR, please:).